nuxeo-features/nuxeo-elasticsearch/nuxeo-elasticsearch-audit/src/test/resources/elasticsearch-audit-index-test-contrib.xml

```diff
@@ -6,31 +6,57 @@
 
     <elasticSearchIndex name="audit" type="entry" >
       <mapping>
-{
-  "_all": {
-    "analyzer": "fulltext"
-  },
-  "dynamic_templates": [
-    {
-      "strings": {
-        "match_mapping_type": "string",
-        "mapping": {
-          "type": "keyword",
-          "ignore_above": 256
+      {
```

Comment:
May be the JSON deserves to have its own file, it will be easier to modify and compare with the version contributed to nuxeo-distribution.